### PR TITLE
docs: Update Free Trial visual

### PR DESCRIPTION
**Pull request by huguestennier**
### PR fulfills these requirements
- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: docs/source/guide/install.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ label-studio
 The default web browser will automatically open at [http://localhost:8080](http://localhost:8080) with Label Studio. Please refer to [start Label Studio](start.html) for more options when starting Label Studio.
 
 !!! note
-    If you've previously installed the brew tap from the now-deprecated organization name `heartexlabs/tap`, we've got good news. You don't have to worry about migrating immediately. The deprecated tap has been set up as a mirror for `humansignal/tap`. This ensures continuity and minimizes disruptions for existing users.
+If you've previously installed the brew tap from the now-deprecated organization name `heartexlabs/tap`, we've got good news. You don't have to worry about migrating immediately. The deprecated tap has been set up as a mirror for `humansignal/tap`. This ensures continuity and minimizes disruptions for existing users.
```
Comment (caitlinwheeless) [inline docs/source/guide/install.md]: @huguestennier - Did you mean to remove the spaces? Doesn't that break the note output? https://deploy-preview-4744--label-studio-docs-new-theme.netlify.app/guide/install#Install-using-Homebrew

Comment (huguestennier) [reply docs/source/guide/install.md]: 🤦‍♂️ Indeed! I have an auto-formatter whenever I save a file and it did that automatically. Fixing now

File: docs/source/guide/install.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ label-studio
...
```
Comment (huguestennier) [inline docs/source/guide/install.md]:
```suggestion
    If you've previously installed the brew tap from the now-deprecated organization name `heartexlabs/tap`, we've got good news. You don't have to worry about migrating immediately. The deprecated tap has been set up as a mirror for `humansignal/tap`. This ensures continuity and minimizes disruptions for existing users.
```

File: docs/source/guide/install.md [hunk 3]
```diff
@@ -255,7 +255,7 @@ Upgrade Docker Engine to the latest available version(>= [20.10.12](https://docs
 ### PermissionError: [Errno 13] Permission denied: `/label-studio/data/media`
 
 !!! warning
-    Starting with Label Studio 1.7.0 release, the application run using a non-root docker user with ID `1001`.
+Starting with Label Studio 1.7.0 release, the application run using a non-root docker user with ID `1001`.
```
Comment (huguestennier) [inline docs/source/guide/install.md]:
```suggestion
    Starting with Label Studio 1.7.0 release, the application run using a non-root docker user with ID `1001`.
```

**Review by caitlinwheeless — APPROVED** [review]

**Pull request merged**